tripal/tripal
# Remove some obsolete files

# Tripal 4 Core Dev Task

### Closes #2046 

## Description
This just removes some files that were left behind as placeholders in PR #2037 because of the reason described in issue #2045 
and the old tripal publish file, which has been replaced with a new publish structure in PR #1991

## Testing?
1. build a docker, removing the placeholder file should have no adverse effect
2. publish even just one record to be confident that the removed file is not needed.
